boilerplate.js

```diff
@@ -43,18 +59,21 @@ function prepare (callback) {
 
   var done = after(this._boilerplate.length, callback)
     , out  = this.boilerplateOut = {}
+    , self = this;
 
-  this._boilerplate.forEach(function (src) {
-    var callback = done
-    findDestination(src, function (err, dst) {
-      if (err)
-        return callback(err)
-
-      out[src] = out[path.basename(src)] = path.basename(dst)
+  this._boilerplate.forEach(function (src, index) {
+    adjustForLang(src, self.lang, function(src) {
+      self._boilerplate[index] = src;
```

Comment:
As I understand it: You store this for performance reasons, correct? For readability of code I would put this at a different place. It hides caching logic in structure. Also I am not sure if that performance is such important as to introduce a possible source of inconsistency.